server/routes/users.js

```diff
@@ -0,0 +1,37 @@
+const express = require("express");
+const router = express.Router();
+const mongoose = require("mongoose");
+
+let databaseConnection = "Waiting for Database response...";
+
+router.get("/", (req, res, next) => {
+    res.send(databaseConnection);
+});
+
+mongoose.connect("mongodb://localhost:27017/mentions", { useNewUrlParser: true  });
+const database = mongoose.connection;
+
+const Schema = mongoose.Schema;
+const ObjectId = Schema.ObjectId;
+
+const UserSchema = new Schema({
+  _id: ObjectId,
+  email: String,
+  password: String,
+  companies: []
+});
+
+const UserModel = mongoose.model('user', UserSchema);
+
```

Comment:
I'd rather those were not here. We want to have a folder just for models instead. You can remove these lines for now, as we just want to connect to the database

Reply:
alright, I removed them